Makefile

```diff
@@ -324,38 +324,38 @@ docker-build-start-cluster:
 	SNAPSHOT=SNAPSHOT_MIX_EXIT_PERIOD_SECONDS_120 make init_test && \
 	docker-compose build --no-cache && docker-compose up
 
-docker-stop-cluster:
+docker-stop-cluster: localchain_contract_addresses.env
```

Comment:
whats the reason these were just added everywhere?

Reply:
It's added wherever we use default  `docker-compose` as it needs to have `localchain_contract_addresses.env` in current dir.

Reply:
why do we need to pull fresh contracts when we `docker-stop-cluster` and again when we start `docker-start-cluster-with-datadog`?

why do we need to pull fresh contracts when we `docker-nuke`?

Reply:
We need to do it only when there is no `localchain_contract_addresses.env`.
Otherwise `docker-compose down` won't work.
We have this section in our `docker-compose.yml`:
```
env_file:
      - ./localchain_contract_addresses.env
```

Reply:
so this checks if the file exists and if it does, it doesn't run the whole `$(MAKE) init-contracts`?

Reply:
yes